src/flows/pipes/Region/view.tsx

```diff
@@ -0,0 +1,301 @@
+import React, {FC, useContext, useEffect, useState, useMemo} from 'react'
+import {useSelector} from 'react-redux'
+
+import {activeFlags} from 'src/shared/selectors/flags'
+import {PipeProp} from 'src/types/flows'
+import {PipeContext} from 'src/flows/context/pipe'
+
+import {
+  Dropdown,
+  ComponentColor,
+  ComponentSize,
+  Form,
+  FlexBox,
+  Input,
+  InputType,
+  ComponentStatus,
+} from '@influxdata/clockface'
+
+import './style.scss'
+
+const REGIONS = [
+  {
+    label: 'US West (Oregon)',
+    group: 'Amazon Web Services',
+    value: 'https://us-west-2-1.aws.cloud2.influxdata.com',
+  },
+  {
+    label: 'US East (Virginia)',
+    group: 'Amazon Web Services',
+    value: 'https://us-east-1-1.aws.cloud2.influxdata.com',
+  },
+  {
+    label: 'Europe West (Frankfurt)',
+    group: 'Amazon Web Services',
+    value: 'https://eu-central-1-1.aws.cloud2.influxdata.com',
+  },
+  {
+    label: 'US Central (Iowa)',
+    group: 'Google Cloud Platform',
+    value: 'https://us-central1-1.gcp.cloud2.influxdata.com',
+  },
+  {
+    label: 'Europe West (Belgium)',
+    group: 'Google Cloud Platform',
+    value: 'https://europe-west1-1.gcp.cloud2.influxdata.com',
+  },
+  {
+    label: 'US East (Virginia)',
+    group: 'Microsoft Azure',
+    value: 'https://eastus-1.azure.cloud2.influxdata.com',
+  },
+  {
+    label: 'Europe West (Amsterdam)',
+    group: 'Microsoft Azure',
+    value: 'https://westeurope-1.azure.cloud2.influxdata.com',
+  },
+  {
+    label: 'Tools Cluster',
+    group: 'Development',
+    flag: 'local-dev',
+    value: 'https://influxdb.aws.influxdata.io',
+  },
+  {
+    label: 'Local Kind Cluster',
+    group: 'Development',
+    flag: 'local-dev',
+    value: 'https://twodotoh.a.influxcloud.dev.local',
+  },
+  {label: 'Self Hosted', value: 'self-hosted'},
+]
```

Comment:
is there any way these can not be in the UI? It just .... stinks. Also, do we want to allow querying of tools?

Reply:
is there an api that provides all of our supported regions? i'd love to use that if it exists.

Reply:
Nope, but maybe there should be?